libs/events/src/types/toastMessages.ts

```diff
@@ -12,7 +12,7 @@ export interface ToastMessagePayloads {
   }
 
   [ToastMessageType.SWAP_POSTED_API]: {
-    orderId: string
+    orderUid: string
```

Comment:
Again, amazing reaction to comments 👏

Reply:
ahahah, is the magic of the waterfall. Something you realize things later and fix right there